ipprovider/ifconfig/ifconfig.go

```diff
@@ -23,10 +23,14 @@ type ifconfigResponse struct {
 }
 
 // New return ifconfig
-func New(c *http.Client) ipprovider.Provider {
+func New(c *http.Client, IPv6 bool) ipprovider.Provider {
```

Comment:
I liked the idea, but what if we will change this a little bit. What do you think about following solution:

In `ipprovider.Provider` interface we are adding new method `ForceIPV6()`

```go
type Provider interface {
	GetIP() (string, error)
	ForceIPV6()
}
```

which will change the behaviour of the provider during setup.

For example for `ifconfig` provider it will look like this:

```go
func (i *ifconfig) ForceIPV6() {
	i.url = "https://v6.ifconfig.co/json"
}
```

Then bootstrap in `main().main.go` will look as bellow:

```go
	// initialize new ipprovider and register IP providers
	provider := ipprovider.New()

	providerList := []ipprovider.Provider{
		ifconfig.New(c),
		ipify.New(c),
		wtfismyip.New(c),
	}

	if cf.ForceIPV6 {
		for _, p := range providerList {
			p.ForceIPV6()
		}
	}

	provider.Register(providerList...)
```

Reply:
Good idea, but some providers (wtfismyip) doesn't support the IPv6/IPv4 forcing.
Or I couldn't find a way yet.

That's why I've added 2 different provider list. :)

Reply:
Yeah, I got your point :)

Hm... 🤔 

In their [documentation](https://wtfismyip.com/why) I see, that they support IPv6. But it will be automatic, I believe.

What JSON do you see here https://wtfismyip.com/json? Do you see some specific field that indicates `IPv6` or it it the same `YourFuckingIPAddress ` field?

Reply:
Nope. :(
```
{
   "YourFuckingIPAddress": "2001:4c4c:****",
   "YourFuckingLocation": "Érd, PE, Hungary",
   "YourFuckingHostname": "****.catv.pool.telekom.hu",
   "YourFuckingISP": "Magyar Telekom",
   "YourFuckingTorExit": "false",
   "YourFuckingCountryCode": "HU"
}
```

Reply:
Oh, I found something:
https://ipv4.wtfismyip.com/json

:)

Reply:
Ok, then for "wtfismyip" we could just implement empty `ForceIPV6` without any logic for now :) 

And indicate in the comment, that this service doing switch automatically